gpu/gpu.py

```diff
@@ -0,0 +1,65 @@
+from random import random
+
+
+def check_squared_value(x, xSquer, testValue):
+    if x == testValue - 1:
+        if x != testValue - 1:
+            return False
+        else:
+            return True
```

Comment:
Te warunki nie mają sensu, jeśli x == testValue - 1 to drugi warunek zawsze będzie niespełniony.

Poza tym, jak w wersji na CPU najpierw sprawdzamy warunek dopiero później potęgujemy, by pominąć niepotrzebne kalkulacje, gdyż wynik nie ma znaczenia przy tym sprawdzaniu

Reply:
Może to błąd przy kopiowaniu i tabowaniu wyszedł jakiś?